server/triple.go

```diff
@@ -75,6 +75,14 @@ func (s *Server) GetTriples(ctx context.Context, in *pb.GetTriplesRequest) (
 			resultsMap[dcid] = applyLimit(dcid, allTriplesCache[dcid].Triples, limit)
 		}
 	}
+	// Regular triples cache from memcache for population data
+	if len(popDcids) > 0 {
+		dataMap := s.memcache.ReadParallel(
+			buildTriplesKey(popDcids), convertTriplesCache)
```

Comment:
Do we have an inventory of which other cache types don't support branch cache?

Reply:
This is something that need thorough examination. We have branch cache for many cache types but I am not entirely sure how are build, and how many overlap they have with base cache. This has indication of the merging. We should find a time later to go through each branch cache to figure out

1) if we need the branch cache
2) how it should be merged.

Reply:
created https://github.com/datacommonsorg/mixer/issues/251 to track

Reply:
Thanks, we can follow up there.